dataland-frontend/tests/e2e/specs/prepopulation/Prepopulation.ts

```diff
@@ -116,5 +118,32 @@ describe(
         checkMatchingIds(DataTypeEnum.EutaxonomyNonFinancials, companiesWithEuTaxonomyDataForNonFinancials.length);
       });
     });
+
+    describe("Upload and validate Lksg data", () => {
```

Comment:
Is it possible to unify the uploading/checking of data for all frameworks? My idea would be to have one function which hast the functionalty and then is run for every framework. This could reduce duplications in the code.

Reply:
We looked into it and the tests are already generic for most of the functionality, the only part that could be improved in my opinion is the loading of the fixture data. I will look into it.

Reply:
It does not add really to the code quality and hece will be skipped.